realm/realm-library/src/objectServer/java/io/realm/mongodb/ErrorCode.java

```diff
@@ -47,6 +47,8 @@
     BSON_CODEC_NOT_FOUND(Type.JAVA, 1100),
     BSON_ENCODING(Type.JAVA, 1101),
     BSON_DECODING(Type.JAVA, 1102),
+    // Java errors
+    RUNTIME_EXCEPTION(Type.JAVA, 1200),
```

Comment:
It helps to catch generic errors as in `RealmEventStreamAsyncTaskImpl.java`

Reply:
As suggested elsewhere it would might be an idea to segment it to indicate a _stream protocol error_ or similar, to make it a bit clearer to reason about